3rdparty/libsvm/tools/README

```diff
@@ -0,0 +1,210 @@
+This directory includes some useful codes:
```

Comment:
@Ky5595 Are we using these Python scripts? If not, then we should not include them in our repo (the license allows modifications to the original source code in any form).